pages/linux/krfb-virtualmonitor.md

```diff
@@ -0,0 +1,8 @@
+# krfb-virtualmonitor
+
+> Create a virtual monitor and allow that monitor to be used with VNC.
+> More information: <https://github.com/KDE/krfb/blob/master/krfb/main-virtualmonitor.cpp>.
```

Comment:
```suggestion
> More information: <https://invent.kde.org/network/krfb>.
```
For now, let's just link to the source repo (btw KDE's GitHub repos are push-only mirrors so I would suggest linking to their GitLab instead as it is proper officially supported place).